packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js

```diff
@@ -42,18 +55,20 @@ storiesOf("Molecules|Select", module)
       },
       errorMessage: {
         default: text("label", "Color", "Props")
```

Comment:
Please name the knob with the property name (`"errorMessage"` not `"label"`).

```suggestion
         default: text("errorMessage", "Color", "Props")
```

Reply:
(Also the other occurrences in this file (3x))

Reply:
Done :)